src/components/custom/_signup-public.js

```diff
@@ -255,6 +256,10 @@ const SignupPublic = ({
     is_submitting,
 }) => {
     const data = useStaticQuery(query)
+    const [checkBoxState, setCheckBoxState] = useState(false)
```

Comment:
Should we name this to something more meaningful like `[is_checked, setChecked]`? We can do it for `_signup-new.js` as well.

Reply:
I was thinking, working on _signup-new.js is out of this task's scope or not. Using the same component will obviously the best option. Thanks for pointing this out.